pkg/module/all_test.go

```diff
@@ -10,6 +10,7 @@ import (
 
 const (
 	// these values need to point to a real repository that has a tag
+	// github.com/NYTimes/gizmo is a example of a path that needs to be encoded so we can cover that case as well
```

Comment:
👍

Reply:
maybe we should create our own upper case repo in github.com/athens-artifacts and write a test with it. Happy to do that as a next pr

Reply:
Great idea @marwan-at-work !